### DISCUSSION TITLE
Docker: set up ejabberd and keep it updated automagically with Watchtower

### DISCUSSION THREAD

**Discussion by mremond** [processone/ejabberd#4311] [Announcements]
Using Docker ?

Here is a step by step tutorial on how to deploy an ejabberd instance and keep it up to date with Watchtower

https://www.process-one.net/blog/docker-ejabberd-watchtower/

#XMPP #Academy #ejabberd

**Comment by poVoq** [comment]
Or just use Podman, which has that integrated.
